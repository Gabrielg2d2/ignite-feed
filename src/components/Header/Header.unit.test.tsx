import { render, screen } from '@testing-library/react'
import { Header } from '.'

describe('Header', () => {
  it('should render snapshot component Header', () => {
    const { container } = render(<Header />)

    expect(container).toMatchSnapshot()
  })

  it('should render component Header with by text Ignite Feed', () => {
    render(<Header />)

    const currentText = screen.getByText(/ignite feed/i)

    expect(currentText).toBeInTheDocument()
  })
})
