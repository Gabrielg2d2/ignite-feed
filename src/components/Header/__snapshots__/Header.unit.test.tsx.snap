// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render snapshot component Header 1`] = `
<div>
  <header
    class="src-components-Header-header-RUooHIpSho"
  >
    <img
      alt="Logotipo Ignite-lab"
      src="ignite-logo.svg"
    />
    <strong>
      Ignite Feed
    </strong>
  </header>
</div>
`;
